assets/css/style.css

```diff
@@ -0,0 +1,708 @@
+  font-family: Lato-Light;
+  src: url(/first-capstone-project/assets/fonts/Lato2OFL/Lato-Light.ttf);
+}
+
+@font-face {
+  font-family: Lato-Bold;
+  src: url(/first-capstone-project/assets/fonts/Lato2OFL/Lato-Bold.ttf);
+}
+
+@font-face {
+  font-family: Lato-Regular;
+  src: url(/first-capstone-project/assets/fonts/Lato2OFL/Lato-Regular.ttf);
+}
+
+@font-face {
+  font-family: Lato-Medium;
+  src: url(/first-capstone-project/assets/fonts/Lato2OFL/Lato-Medium.ttf);
+}
+
+@font-face {
+  font-family: Lato-BoldItalic;
+  src: url(/first-capstone-project/assets/fonts/Lato2OFL/Lato-BlackItalic.ttf);
+}
+
+@font-face {
+  font-family: Lato-Italic;
+  src: url(/first-capstone-project/assets/fonts/Lato2OFL/Lato-Italic.ttf);
+}
+
+@font-face {
+  font-family: Cocogoose-Pro-Block-Innerline;
+  src: url(/first-capstone-project/assets/fonts/cocogoose-pro/Cocogoose-Pro-Block-Innerline.ttf);
+}
+
+@font-face {
+  font-family: Cocogoose-Pro-Light;
+  src: url(/first-capstone-project/assets/fonts/cocogoose-pro/Cocogoose-Pro-Light.ttf);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Lato-Regular', sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:visited {
+  text-decoration: none;
+  color: #ec5242;
+}
+
+body {
+  background-color: #f5f5f5;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+ul {
+  list-style-type: none;
+}
+
+footer img {
+  width: 11rem;
+}
+
+.navbar > img {
+  width: 24px;
+}
+
+.desktop {
+  display: none;
+}
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+  background-image: url(/first-capstone-project/assets/images/bg2.png);
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+}
+
+.socials {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.socials:hover,
+.userlinks:hover {
+  cursor: pointer;
+}
+
+.hero {
+  display: flex;
+  opacity: 0.7;
+  flex-direction: column;
+  gap: 20px;
+  background-color: #f5f5f5;
+  margin-top: 4rem;
+}
+
+.hero img {
+  width: 100px;
+}
+
+.hero h1 {
+  text-align: start;
+  text-transform: uppercase;
+  font-family: 'Lato-Bold', sans-serif;
+  font-size: 2rem;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  margin-top: 2rem;
+  font-family: 'Lato', sans-serif;
+  color: #f81404;
+}
+
+.headline-about {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 2rem;
+  font-family: 'Lato', sans-serif;
+  color: #f81404;
+}
+
+.headline > p {
+  font-size: 1.5rem;
+  margin-bottom: 1rem;
+}
+
+.headline-about > p {
+  font-size: 1.3rem;
+  margin-bottom: 1rem;
+}
+
+.headline h1 {
+  text-transform: uppercase;
+  color: #f81404;
+}
+
+.title-h1 {
+  background: url(/first-capstone-project/assets/images/title-bg.jpg);
+  background-position: center;
+  background-size: cover;
+  background-repeat: repeat;
+  -webkit-background-clip: text;
+  /* stylelint-disable-next-line csstree/validator */
+  -webkit-text-fill-color: transparent;
+}
+
```

Comment:
- [ ] While you were making the changes the previous reviewer suggested. I noticed your title alignment has changed so please make sure that your title is aligned left as it is in the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).  :pray: 

  ### Your design
  ![image](https://github.com/NoelLincoln/first-capstone-project/assets/92303891/6c8ee1bf-127b-4fc8-b3e2-4f9de6d2c581)

  ### Cindy's Design
  ![image](https://github.com/NoelLincoln/first-capstone-project/assets/92303891/ca5826e2-0c8d-43ed-a172-060413078edc)